rpcserver.go

```diff
@@ -1268,12 +1268,34 @@ func (s *rpcServer) SubmitOrder(ctx context.Context,
 
 	rpcLog.Infof("New order submitted: nonce=%v, type=%v", o.Nonce(), o.Type())
 
+	// We should update the sidecar ticket if this was a bid created for it.
+	// The ticket in the order struct was already updated with the order
+	// information and signature, but it was only stored as TLV data with
+	// the order itself. We should also update the ticket in the main
+	// sidecar ticket bucket.
+	var encodedTicket = ""
+	bid, isBid := o.(*order.Bid)
+	if isBid && bid.SidecarTicket != nil {
+		err := s.server.db.UpdateSidecar(bid.SidecarTicket)
```

Comment:
Need to double check here to ensure this doesn't violate any assumptions w/ the server-side itests.

Reply:
This is basically just a shortcut. Before you had to query the orders again to get the updated ticket, now you get it in the response directly. There is an itest for this PR as well, btw.